waiter/integration/waiter/token_request_test.clj

```diff
@@ -267,7 +307,9 @@
                                        :owner (retrieve-username)
                                        :run-as-user "i-do-not-exist-but-will-not-be-checked"
                                        :token token}
-                    response (post-token waiter-url token-description :query-params {"update-mode" "admin"})]
```

Comment:
We never create tokens in admin mode.  The test will be better when we structured it to:
1. Create a token normally
2. Get the token data and ETag
3. Update the token using the ETag

Reply:
We should also test a token update with a stale ETag..